packages/oc-template-es6-compiler/lib/viewTemplate.js

```diff
@@ -0,0 +1,14 @@
+const viewTemplate = ({ css, bundle }) => `function(model){
+  var template = ${bundle};
+  return '' + 
+    template.default(model) +
+    '<script>' +
+      'window.oc = window.oc || {};' +
+      'oc.cmd = oc.cmd || [];' +
+      'oc.cmd.push(function(oc){' +
+        'oc.addStylesToHead(\\'${css}\\');' +
```

Comment:
should we move to <style> + event to keep this consistent with the React one?

Reply:
yepa